spec/v2/index.bs

```diff
@@ -2184,7 +2185,18 @@ path: LICENSE.md
 # Appendix B: Changelog # {#changelog}
 
 
-## Version 2.0.1-20230314 (Mar 1, 2023) ## {#changelog-2.0.1-20230314}
+## Version 2.0.1-20230515 (May 15, 2023) ## {#changelog-2.0.1-20230515}
+
+This version fixes 1 definition incorrectness and includes 3 documentation improvements.
+
+1. property <{CarbonFootprint/biogenicCarbonContent}>: was incorrectly
```

Comment:
Could you double check, because unless I'm mistaken I think the change in this PR made to `dLucGhgEmissions` rather than `biogenicCarbonContent`

Reply:
(yap. see above)